### ISSUE TITLE
Attempting to attach a system clock monitor fails on Linux

### ISSUE THREAD

**Issue by gabrielschulhof** [solettaproject/soletta#1350]
When calling ```sol_platform_add_system_clock_monitor()``` I get
```
WRN:sol-platform ./src/lib/common/sol-platform-linux-common.c:844 sol_platform_register_system_clock_monitor() Could not register a timer to watch for system_clock changes.
WRN:sol-platform ./src/lib/common/sol-platform.c:552 monitor_add_and_register() r (-22) < 0
```
I'm running Fedora 21 32-bit with kernel 4.1.13-100.fc21.i686.

**Comment by barbieri** [comment]
32bits issue. Thanks for pointing.

Check proposed fix in PR #1354

**Comment by cabelitos** [comment]
The fix is not merged. Check: fc2efe4af2476488a62b85216aa08e072b0cae6f
